src/m365/spo/commands/contenttype/contenttype-add.ts

```diff
@@ -68,16 +70,30 @@ class SpoContentTypeAddCommand extends SpoCommand {
 
         return request.post(requestOptions);
       })
-      .then((res: string): void => {
+      .then((res: string): Promise<void> => {
         const json: ClientSvcResponse = JSON.parse(res);
         const response: ClientSvcResponseContents = json[0];
         if (response.ErrorInfo) {
           cb(new CommandError(response.ErrorInfo.ErrorMessage));
-          return;
+          return Promise.resolve();
         }
-        
-        cb();
-      }, (err: any): void => this.handleRejectedPromise(err, logger, cb));
+
+        const options: SpoContentTypeGetCommandOptions = {
+          webUrl: args.options.webUrl,
+          listTitle: args.options.listTitle,
+          id: args.options.id,
+          debug: this.debug,
+          verbose: this.verbose
+        };
+        return Cli.executeCommand(SpoContentTypeGetCommand as Command, { options: { ...options, _: [] } });
+      }, (err: any): void => this.handleRejectedPromise(err, logger, cb))
+      .then(_ => cb(), (err: any): void => {
```

Comment:
On line 88 you retrieve the content type, but now we also need to log it to the output. Also, we should use `Cli.executeCommandWithOutput` so that we can capture the output of the command (which is the information about the content type) and send it to logger to pass it to stdout. Just executing the command will retrieve the content type, but will not log it anywhere.

Reply:
a ok. I did not notice it. Thanks for the advice :). Will change

Reply:
ok I changed to executeCommandWithOutput and log the response. Please let me know if this are the changes you were expecting ?